Tutorials/src/mcHistograms.cxx

```diff
@@ -92,19 +92,25 @@ struct AccessMcTruth {
 // For each of them get the corresponding tracks
 struct LoopOverMcMatched {
   OutputObj<TH1F> etaDiff{TH1F("etaDiff", ";eta_{MC} - eta_{Rec}", 100, -2, 2)};
-  void process(aod::McCollision const& mcCollision, soa::Join<aod::McCollisionLabels, aod::Collisions> const& collisions,
+  void process(aod::McCollisions const& mcCollisions, soa::Join<aod::McCollisionLabels, aod::Collisions> const& collisions,
                soa::Join<aod::Tracks, aod::McTrackLabels> const& tracks, aod::McParticles const& mcParticles)
   {
-    // access MC truth information with mcCollision() and mcParticle() methods
-    LOGF(info, "MC collision at vtx-z = %f with %d mc particles and %d reconstructed collisions", mcCollision.posZ(), mcParticles.size(), collisions.size());
-    for (auto& collision : collisions) {
-      LOGF(info, "  Reconstructed collision at vtx-z = %f", collision.posZ());
+    auto collision = collisions.begin();
+    for (auto& mcCollision : mcCollisions) {
+      // access MC truth information with mcCollision() and mcParticle() methods
+      LOGF(info, "MC collision at vtx-z = %f with %d mc particles and %d reconstructed collisions", mcCollision.posZ(), mcParticles.size(), collisions.size());
```

Comment:
I don't see this for mcParticles. Your PR changes the printout of the number of mcParticles *per* collision to a printout of the total length of the mcParticles table. Am I missing something?